### ISSUE TITLE
Reactor keeps offline

### ISSUE THREAD

**Issue by vr6syncro** [Kasra-G/ReactorController#2]
Hey, thanks for the great work.

I have a problem that the Reaktop stays "offline" it says and displays online but it is not producing any rf. The fuel rods are 100% down.
It is only working with the calibration on the first run. Is this finished the reactor stopps working again. ( Fuel Rod goes down to 100%)

ATM6 16.5.x

Hope u can help with that.

**Comment by Kasra-G** [comment]
Hey, what mod and version are you using that adds the reactor?

**Comment by vr6syncro** [comment]
Hey i updated to AllTheMods 6-1.8.15 to see if this makes a different but id doesn't. I am using Bigger Reactor biggerreactors-1.16.5-0.5.1

Thanks for your quick reply.

**Comment by Kasra-G** [comment]
I've done a few tests and the only way the reactor could be online but not generating power is when the internal power buffer is greater than the max buffer percentage. Typically this can mean a few things:

- The reactor's energy buffer is full or too high
- Your "Buffer Target Range" is too low
- You are not draining enough power

The program attempts to keep the energy buffer within a range. By default the energy buffer of the reactor will stay between 30% and 70% full.  If the power in the buffer exceeds the maximum percent, the control rods will go to 100% in an attempt to not waste fuel. 

**TLDR: Make sure you are draining the reactor's power or else it will keep the control rods at 100% to save fuel**

Let me know if this helps.

**Comment by vr6syncro** [comment]
Thank you for the help. That was the Problem. The Buffer was to high and i didn't know how it interacts with the computer.

Thanks, its now working.
